lib/tasks/configuration.rake

```diff
@@ -49,7 +49,7 @@ def test_smtp
   authtype = ENV['SMTP_AUTH'].present? && ENV['SMTP_AUTH'] != "none" ? ENV['SMTP_AUTH'] : nil
 
   smtp.start(ENV['SMTP_DOMAIN'], user, password, authtype) do |s|
-    s.sendmail('test', ENV['SMTP_USERNAME'], 'notifications@example.com')
+    s.sendmail('test', ENV['SMTP_USERNAME'], ENV['SMTP_TEST_RECIPIENT'])
```

Comment:
Would be better to replace this with `ENV.fetch('SMTP_TEST_RECIPIENT', 'notifications@example.com')` so that it defaults incase `SMTP_TEST_RECIPIENT` is not set

Reply:
Ok! changed according to your recommendation